### ISSUE TITLE
Careturn app logo is not displaying correctly

### ISSUE THREAD

**Issue by yeehowe** [abbasali-io/careturn-issues#2]
![IMG_0545](https://github.com/user-attachments/assets/ace67306-3e69-4141-986e-e43411b0aeee)

The android phone font setting is set to bigger size, is this the reason the app icon is not display properly?

**Comment by abbasali-io** [comment]
@yeehowe this is the issue that we discussed earlier. You have already been advised about it. the logo is TOO wordy, just look at this screen for reference, how many 'too long name' logo you see, there is only so much space available for an icon. if we make it too small, the words will not be legible.

I am going to close this issue, if Careturn decide to use a logo, please create a new issue, or re-open this issue and provide a 512x512 PNG logo.
